phpBB/adm/style/converter_list.html

```diff
@@ -0,0 +1,16 @@
+{{ include('installer_header.html') }}
+<h1>{TITLE}</h1>
+<p>{BODY}</p>
+
+<div class ="yaml_list" style=" ">
```

Comment:
Remove the space between `class` and `=`. What is the reason for this `style=" "` here?